DEPS

```diff
@@ -39,42 +39,62 @@ Device detection:
 
 CD info:
 
-* python-cddb
+* cddb (python2)
```

Comment:
We're using [this one](http://cddb-py.sourceforge.net/), right? I recommend keeping it as "python-cddb" as that's the package name on most distros, maybe add the URL as well for clarity. Just "cddb" I think is too confusing because there are so many projects with similar names.

Reply:
I hope to drop the cddb dependency soon anyway, see #608.